DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -163,4 +124,37 @@
   <class name="reco::ClusterRemovalInfo::SiPixelClusterRefProd" />
   <class name="reco::ClusterRemovalInfo::SiStripClusterRefProd" />
   <class name="edm::Wrapper<reco::ClusterRemovalInfo>" />
+
+  // fastsim containers
+  
+  <class name="FastTrackerRecHit" ClassVersion="10">
```

Comment:
hi slava

I would be very surprised if there were any interest in running with 76 on samples with older releases. But I could put the old classes_def lines back if that is desirable and technically possible.
(I have no clue how CMSSW will react having lines for classes that no longer exist)